Packs/Armis/Integrations/Armis/Armis.yml

```diff
@@ -0,0 +1,491 @@
+category: Utilities
+commonfields:
+  id: Armis
+  version: -1
+configuration:
+- display: Server URL
+  name: url
+  required: true
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- defaultvalue: '10'
+  display: Maximum number of incidents per fetch
+  name: max_fetch
+  required: false
+  type: 0
+- defaultvalue: ACTIVE
+  display: Fetch alerts with status (UNHANDLED,SUPPRESSED,RESOLVED)
+  name: alert_status
+  options:
+  - UNHANDLED
+  - SUPPRESSED
+  - RESOLVED
+  required: false
+  type: 16
+- additionalinfo: The types of Alerts are Policy Violation, System Policy Violation,
+    Anomaly Detection
```

Comment:
If no type is chosen then all types are being fetched? if so, please mention it.